docs/common/DocsExample.vue

```diff
@@ -123,6 +123,36 @@
         default: false,
         required: false,
       },
+      /**
+       * If true, hides the template (HTML) code block from the code viewer.
+       * @type {Boolean}
+       * @default false
+       */
+      hideTemplate: {
+        type: Boolean,
+        default: false,
+        required: false,
+      },
+      /**
+       * If true, hides the script (JavaScript) code block from the code viewer.
+       * @type {Boolean}
+       * @default false
+       */
+      hideScript: {
+        type: Boolean,
+        default: false,
+        required: false,
+      },
+      /**
+       * If true, hides the style (SCSS) code block from the code viewer.
```

Comment:
Let's change this to simply "Hides the style (SCSS) code block from the code viewer."